coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/provider/util/JsonbUtil.java

```diff
@@ -55,25 +61,86 @@ public class JsonbUtil {
+    private static final String SERIALIZERS_POSTFIX = "serializers";
+    private static final String DESERIALIZERS_POSTFIX = "deserializers";
+
     /**
      * Prefix for all configs
      */
     public static final String JSONB_CONFIG_PREFIX = "coffee.jsonb.config";
 
-    /**
-     * Create {@link Jsonb} instance with {@link FieldOnlyVisibilityStrategy} property visibility strategy.
-     * 
-     * @return configured {@link Jsonb} instance
-     */
-    public static Jsonb getContext() {
+    private static final Jsonb JSONB;
+
+    static {
         Config config = ConfigUtil.getInstance().defaultConfig();
         JsonbConfig jsonbConfig = new JsonbConfig()
                 // property visibility strategy setting
                 .withPropertyVisibilityStrategy(getPropertyVisibilityStrategyClass(config))
                 .withBinaryDataStrategy(getBinaryDataStrategy(config));
-        return JsonbBuilder.newBuilder().withConfig(jsonbConfig).build();
+
+        getStringConfig(config, BINARY_DATA_STRATEGY_POSTFIX)
+                .ifPresent(jsonbConfig::withBinaryDataStrategy);
+
+        getBooleanConfig(config, FORMATTING_POSTFIX)
+                .ifPresent(jsonbConfig::withFormatting);
+
+        getBooleanConfig(config, NULL_VALUES_POSTFIX)
+                .ifPresent(jsonbConfig::withNullValues);
+
+        getStringConfig(config, ENCODING_POSTFIX)
+                .ifPresent(jsonbConfig::withEncoding);
+
+        getBooleanConfig(config, STRICT_IJSON_POSTFIX)
+                .ifPresent(jsonbConfig::withStrictIJSON);
+
+        getPropertyNamingStrategyClass(config)
+                .ifPresent(jsonbConfig::withPropertyNamingStrategy);
+
+        getStringConfig(config, PROPERTY_ORDER_STRATEGY_POSTFIX)
+                .ifPresent(jsonbConfig::withPropertyOrderStrategy);
+
+        getStringConfig(config, DATE_FORMAT_POSTFIX)
+                .ifPresent(dateFormat -> jsonbConfig.withDateFormat(dateFormat, getLocaleConfig(config).orElse(Locale.getDefault())));
+
+        getLocaleConfig(config)
+                .ifPresent(jsonbConfig::withLocale);
+
+        // Apply adapters, serializers, and deserializers only if they are configured
+        JsonbAdapter[] adapters = getAdapters(config);
+        if (adapters.length > 0) {
+            jsonbConfig.withAdapters(adapters);
+        }
+
+        JsonbSerializer[] serializers = getSerializers(config);
+        if (serializers.length > 0) {
+            jsonbConfig.withSerializers(serializers);
+        }
+
+        JsonbDeserializer[] deserializers = getDeserializers(config);
+        if (deserializers.length > 0) {
+            jsonbConfig.withDeserializers(deserializers);
+        }
+
+        JSONB = JsonbBuilder.newBuilder().withConfig(jsonbConfig).build();
+    }
+
+    /**
+     * {@link Jsonb} instance getter.
+     *
+     * @return configured {@link Jsonb} instance
+     */
+    public static Jsonb getContext() {
+        return JSONB;
```

Comment:
a 155.sorban (csak nem engedte kommentelni) jo lenne kiiratni magat a hiba okat is.

Reply:
ezt a részt átírtuk ebben a pull requestben https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/pull/702/files#diff-b36c3e6207f73415def2a553a995e41a13735c7c0d71636022f4f8fc1b3bae70R283